src/garth/data/body_battery/events.py

```
@@ -0,0 +1,149 @@
+from datetime import date, datetime, timedelta
```

Ruff (0.11.9)

`datetime.timedelta` imported but unused

Remove unused import: `datetime.timedelta`

(F401)